<map id="pytest_userver.plugins.log_capture.IncorrectUsageError" name="pytest_userver.plugins.log_capture.IncorrectUsageError">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,167,216,207"/>
<area shape="rect" id="Node000002" href="$df/d4b/classpytest__userver_1_1plugins_1_1log__capture_1_1BaseError.html" title=" " alt="" coords="14,79,207,119"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="113,133,113,166,108,166,108,133"/>
<area shape="rect" id="Node000003" title=" " alt="" coords="67,5,154,31"/>
<area shape="poly" id="edge2_Node000002_Node000003" title=" " alt="" coords="113,44,113,79,108,79,108,44"/>
</map>
